### PR TITLE
[JENKINS-64805] Fixes IE11 error on the available tab

### PR DESCRIPTION
**Pull request by fqueiruga**
See [JENKINS-64805](https://issues.jenkins-ci.org/browse/JENKINS-64805).

https://github.com/jenkinsci/jenkins/pull/5051 introduced an issue on IE11 where the _available_ tab of the plugin manager would not work at all due to the use of `Array.includes` throwing an error. This PR fixes it.

Captura de pantalla 2021-02-04 a las 17 30 43

This is a big regression and a LTS-candidate IMO.

### Proposed changelog entries

* Fix issue where the _available_ plugins tab would break on IE11.

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@timja 
@MRamonLeon 
@alecharp 
@oleg-nenashev 

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

**Review by alecharp — APPROVED** [review]

File: war/src/main/js/plugin-manager-ui.js
Comment (timja) [inline war/src/main/js/plugin-manager-ui.js:37]: could we just polyfill includes?

Comment (fqueiruga) [reply war/src/main/js/plugin-manager-ui.js:37]: We should, but for a quick regression fix I'd rather keep it simple and follow up with proper work.

**Review by timja — APPROVED** [review]

**Review by res0nance — APPROVED** [review]

File: war/src/main/js/plugin-manager-ui.js
Comment (MRamonLeon) [reply war/src/main/js/plugin-manager-ui.js:37]: @fqueiruga is => working on IE11? I thought it doesn't work https://stackoverflow.com/a/40216100/4728711

**Review by MRamonLeon — DISMISSED** [review]
Could you confirm @fqueiruga the arrow works on IE11? I would swear it doesn't work: https://caniuse.com/arrow-functions

**Comment by fqueiruga** [comment]
I have tested it, it does. Arrow functions are transpiled to normal functions by Babel. Babel is transpiling many modern language constructs such as `const`, `let`, arrow functions, etc. What it does not is add polyfills for unsupported APIs, which was the issue here (unsupported `Array.includes`).

File: war/src/main/js/plugin-manager-ui.js
Comment (fqueiruga) [reply war/src/main/js/plugin-manager-ui.js:37]: Yes, it works

**Comment by timja** [comment]
This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Review by MRamonLeon — APPROVED** [review]

**Review by oleg-nenashev — COMMENTED** [review]
Belated +1 for the fix and backporting

**Pull request merged**
